ktor-demo/app_base/src/model/WhoIsResponse.kt

```diff
@@ -0,0 +1,9 @@
+package com.joram.model
+
+import com.fasterxml.jackson.annotation.JsonProperty
+
+data class WhoIsResponse(
+    @JsonProperty("requestFrom") val requestFrom: String,
+    @JsonProperty("responseFrom") val responseFrom: String,
```

Comment:
Maybe add something like communicationProtocol? (For now only using HTTP, but might want to try out gRPC and more.)